CHANGELOG.md

```diff
@@ -1,4 +1,96 @@
-## 2.18.2 (Unreleased)
+## 2.19.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.4.2416**
+- Supported Terraform version: **v0.12.x**, **v0.13.x** and **v0.14.x**
+
+### Features:
+1. Implemented support of following attributes in **aviatrix_spoke_gateway**:
+  - ``enable_private_vpc_default_route``
+  - ``enable_skip_public_route_table_update``
+  - ``enable_auto_advertise_s2c_cidrs``
+2. Implemented support of associating a controller with a CoPilot instance:
+  - New resource **aviatrix_copilot_association**
+3. Implemented support for enabling event triggered HA for Site2Cloud type connection resources:
+  - New attribute ``enable_event_triggered_ha`` in **aviatrix_site2cloud**, **aviatrix_transit_external_device_conn**, **aviatrix_vgw_conn** and **aviatrix_device_transit_gateway_attachment**
+4. Implemented support for GCP FireNet with Fortinet and CheckPoint firewall vendors
+5. Implemented support for Insane Mode over Internet with following attributes in **aviatrix_transit_gateway_peering**:
+  - ``enable_insane_mode_encryption_over_internet``
+  - ``tunnel_count``
+6. Implemented support for attaching a managed CloudN device to an **aviatrix_transit_gateway**:
+  - New resource **aviatrix_cloudn_transit_gateway_attachment**
+7. Implemented support for setting approved CIDRs in **aviatrix_transit_external_device_conn**:
+  - New attribute ``approved_cidrs``
+8. Implemented support for remote syslog profile name:
+  - New attribute ``name`` in **aviatrix_remote_syslog**
+9. Implemented new resources to decouple ``security_domains`` out of **aviatrix_aws_tgw**:
+  - **aviatrix_aws_tgw_security_domain**
+  - **aviatrix_aws_tgw_security_domain_connection**
+10. Implemented support for intra domain inspection:
+  - New resource **aviatrix_aws_tgw_intra_domain_inspection**
+11. Implemented support for multi-tier transit feature:
+  - New attribute ``enable_multi_tier_transit`` in **aviatrix_transit_gateway** and data source
+12. Implemented support for Alibaba Cloud in resources **aviatrix_account**, **aviatrix_vpc**, **aviatrix_gateway**, **aviatrix_transit_gateway** and **aviatrix_spoke_gateway** and data sources
+13. Implemented support for disabling sending exception emails to Aviatrix:
+  - New resource **aviatrix_controller_email_exception_notification_config**
+14. Implemented support for updating certificate domain:
+  - New resource **aviatrix_controller_cert_domain_config**
+15. Implemented support for TGW segmentation for Egress:
+  - New attribute ``tgw_segmentation_for_egress_enabled`` in **aviatrix_firenet** and data source
+16. Implemented support for OCI FireNet
+17. Implemented support for Egress FireNet route injection:
+  - New attribute ``egress_static_cidrs`` in **aviatrix_firenet** and data source
+18. Implemented support for launching user provided AMI for FireNet:
```

Comment:
18. Implemented custom AMI support for Firewall instance, allowing customers to launch special images provided by firewall vendors: